css/index.css

```diff
@@ -18,12 +18,16 @@ menu, nav, output, ruby, section, summary,
 time, mark, audio, video {
```

Comment:
I see you added all of your CSS on the same page. Since you ended up creating about 4 pages I would recommend having more than one CSS page.